core/variant_call.cpp

```diff
@@ -35,1058 +35,729 @@
 #include "core/crypto/crypto_core.h"
 #include "core/debugger/engine_debugger.h"
 #include "core/io/compression.h"
+#include "core/oa_hash_map.h"
 #include "core/object.h"
 #include "core/os/os.h"
 
+_FORCE_INLINE_ void sarray_add_str(Vector<String> &arr) {
+}
```

Comment:
Is this actually used? Or is it something necessary to make the template magic work?